dash-pipeline/Makefile

```diff
@@ -176,10 +176,6 @@ sai-headers: p4 docker-saithrift-bldr-image-exists | SAI/SAI
 
 sai-meta:
 	@echo "Generate SAI metadata..."
-	# hack - remove scripts which cause Git ownership failures in CI pipelines
-	# We don't need them, they're to check that SAI headers didn't experience enum changes etc.
-	sed -ie '/checkenumlock.sh/d' SAI/SAI/meta/Makefile
-	sed -ie '/checkancestry.sh/d' SAI/SAI/meta/Makefile
```

Comment:
this hack is no longer needed, i updated SAI/meta https://github.com/opencomputeproject/SAI/pull/1833 since size.sh have similar issue, but it must generate size file